graphrag/index/create_pipeline_config.py

```diff
@@ -332,37 +259,103 @@ def _graph_workflows(
 
 
 def _community_workflows(
-    settings: GraphRagConfig, covariates_enabled: bool, embedded_fields: set[str]
+    settings: GraphRagConfig, covariates_enabled: bool
 ) -> list[PipelineWorkflowReference]:
-    skip_community_title_embedding = community_title_embedding not in embedded_fields
-    skip_community_summary_embedding = (
-        community_summary_embedding not in embedded_fields
-    )
-    skip_community_full_content_embedding = (
-        community_full_content_embedding not in embedded_fields
-    )
     return [
         PipelineWorkflowReference(name=create_final_communities),
         PipelineWorkflowReference(
             name=create_final_community_reports,
             config={
                 "covariates_enabled": covariates_enabled,
-                "skip_title_embedding": skip_community_title_embedding,
-                "skip_summary_embedding": skip_community_summary_embedding,
-                "skip_full_content_embedding": skip_community_full_content_embedding,
                 "create_community_reports": {
                     **settings.community_reports.parallelization.model_dump(),
                     "async_mode": settings.community_reports.async_mode,
                     "strategy": settings.community_reports.resolved_strategy(
                         settings.root_dir
                     ),
                 },
+            },
+        ),
+    ]
+
+
+def _covariate_workflows(
+    settings: GraphRagConfig,
+) -> list[PipelineWorkflowReference]:
+    return [
+        PipelineWorkflowReference(
+            name=create_final_covariates,
+            config={
+                "claim_extract": {
+                    **settings.claim_extraction.parallelization.model_dump(),
+                    "strategy": settings.claim_extraction.resolved_strategy(
+                        settings.root_dir, settings.encoding_model
+                    ),
+                },
+            },
+        )
+    ]
+
+
+def _embeddings_workflows(
+    settings: GraphRagConfig, embedded_fields: set[str]
+) -> list[PipelineWorkflowReference]:
+    return [
+        PipelineWorkflowReference(
+            name=generate_text_embeddings,
+            config={
+                "document_raw_content_embed": _get_embedding_settings(
+                    settings.embeddings,
+                    "document_raw_content",
+                    {
+                        "title_column": "raw_content",
+                        "collection_name": "final_documents_raw_content_embedding",
+                        "snapshot": settings.snapshots.embeddings,
```

Comment:
Move these to a single top-level config just like the graphml one (it can be called embeddings_snapshot to keep the same pattern). That way it does not need to be repeated, nor does it need to be dug out of the deep config later in the flow.